src/landmark.c

```diff
@@ -8,21 +11,21 @@ struct Landmark
 
 struct LandmarkList
 {
-    u8 field_0;
+    u8 mapSection;
     u8 field_1;
     u16 field_2;
     
-    struct Landmark **landmark_list;
+    struct Landmark **landmarks;
 };
 
 extern const struct LandmarkList gLandmarkLists[];
 
-const struct Landmark **GetLandmarkList(u8 arg_0, u8 arg_1);
+static const struct Landmark **GetLandmarks(u8 arg_0, u8 arg_1);
 
 u8 *GetLandmarkName(u8 arg_0, u8 arg_1, u8 count)
```

Comment:
I tried to rewrite this one and make it more readable, but I couldn't get it to match.